.github/workflows/go-coverage.yml

```diff
@@ -0,0 +1,29 @@
+name: Test coverage
+on:
+  push:
+    tags:
+      - v*
```

Comment:
I think we can remove the `tags` section, I'm not sure we care if this runs on tags